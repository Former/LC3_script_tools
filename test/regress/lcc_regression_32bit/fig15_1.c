#include <stdio.h>

int main()
{
   int i;
   int j;

   for (i = 0; i <= 10; i++) {
       j = i * 76543;
       printf("%d x 76543 = %d\n", i, j);
   }
  return 0;
}
