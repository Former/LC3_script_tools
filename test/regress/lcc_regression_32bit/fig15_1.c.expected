0 x 76543 = 0
1 x 76543 = 76543
2 x 76543 = 153086
3 x 76543 = 229629
4 x 76543 = 306172
5 x 76543 = 382715
6 x 76543 = 459258
7 x 76543 = 535801
8 x 76543 = 612344
9 x 76543 = 688887
10 x 76543 = 765430
